### DISCUSSION TITLE
Add cheat to make guests ignore price limits

### DISCUSSION THREAD

**Discussion by MarcelVos96** [OpenRCT2/OpenRCT2#21669] [Feature requests]
Similar to the "guests ignore intensity" cheat I think it'd be useful to have a cheat that makes guests ignore prices. They will all go on a $15 merry-go-round, and no-one has any qualms about buying a lemonade that costs $17.50.

This would've been useful for me a bunch of times over the past few months, and I'm sure others would like it as well.

**Comment by CarterMoody** [comment]
I like this idea. Would there be any reason to have guests ignore price, and not intensity? I ask because would a "ignore all deterrents" button satisfy this? Or is there a use case where one would like to pick and choose which facets of a ride that a guest ignores? Lastly: Prices, intensities... are there any other factors that will cause a guest to ignore a ride that could be also added to the cheat menu? (ignoring a ride because G forces have not been calculated might be one example).

**Reply by MarcelVos96** [reply]
"Ignore all deterrents" is a bit vague and unclear for people who don't already know what it does. Also for my current project this cheat would've been useful for I specifically don't want to ignore intensity.

There is also the "ride is unsafe" deterrent, but there is already a button in the cheat menu that resets that status for all rides.

**Comment by sjoerddebruin** [comment]
This has been added in #21675.
